plugins/chat/assets/javascripts/discourse/initializers/chat-sidebar.js

```diff
@@ -74,12 +73,20 @@ export default {
             }
 
             get suffixValue() {
-              return chatChannelsManager.publicMessageChannels.some(
+              return chatChannelsManager.allChannels.some(
                 (channel) => channel.unreadThreadsCount > 0
               )
                 ? "circle"
                 : "";
             }
+
+            get suffixCSSClass() {
```

Comment:
Allows green dot next to My Threads on chat sidebar when a watched thread has unreads.